gdx/src/com/badlogic/gdx/utils/Queue.java

```diff
@@ -300,8 +316,8 @@ public T get (int index) {
 		return values[i];
 	}
 
-	/** Removes all values from this queue. Values in backing array are set to null to prevent memory leak, so this operates in
```

Comment:
looks like you formatted manually (newline), please only format lines you changed.